src/tigerasi/tiger_controller.py

```diff
@@ -364,7 +374,22 @@ def set_axis_backlash(self, wait: bool = True, **axes: float):
             the absolute position (in steps) to move to.
         :param wait: wait until the reply has been received.
         """
-        self._set_cmd_args_and_kwds(Cmds.BACKLASH, **axes, wait=wait)
+        return self._set_cmd_args_and_kwds(Cmds.BACKLASH, **axes, wait=wait)
+
+    @axis_check()
+    def get_axis_backlash(self, *axes: str):
+        """Return the backlash compensation value for one or more axes.
+
+        :param axes: one or more lettered axes (case insensitive).
+        :return: backlash compensation of requested axes in dict form (upper case).
+
+        .. code-block:: python
+
+            box.get_backlash_compensation('x')  # returns: {'X': 0.1}
```

Comment:
It looks like this comment is out of date and should probs be `get_axis_backlash`